s3repo/model.py

```diff
@@ -93,9 +94,22 @@ def sync(self):
                     str(self.s3_settings['endpoint_url']),
                     '--s3-region',
                     str(self.s3_settings['region']),
-                    's3://' + self.s3_settings['bucket_name'] + '/' + sync_repo
```

Comment:
Nit: Why do you moved it? Now it looks as the whole command on the first glance, but it does not. If we need to form the command step by step, it would be better to do it as a series of `.append()`s: so it'll be obvious that it is not the whole command.

Reply:
I see now: we append `--sign` conditionally. Overlooked it on the first glance.

Reply:
> Nit: Why do you moved it? Now it looks as the whole command on the first glance, but it does not. If we need to form the command step by step, it would be better to do it as a series of `.append()`s: so it'll be obvious that it is not the whole command.

If I rewrite all the arguments with append, it looks like unnecessary changes in the commit.

Reply:
I don't insist here.